packages/wrangler/src/api/inspect.ts

```diff
@@ -140,16 +140,16 @@ export type DtEvent = DtLogEvent | DtExceptionEvent;
  */
 export type DtListener = (event: DtEvent) => void;
 
-interface DtProtocolRequest<T> {
-  id: number;
-  method: string;
-  params?: T;
-}
-
-interface DtProtocolResponse<T> {
-  id: number;
-  result: T;
-}
+// interface DtProtocolRequest<T> {
+//   id: number;
+//   method: string;
+//   params?: T;
+// }
+
+// interface DtProtocolResponse<T> {
+//   id: number;
+//   result: T;
+// }
```

Comment:
Yeah I should, why not. It's a leftover from the early prototype, and I figured it may be useful later when we recator this whole thing but... eh, we can always recover it. Deleting.